style.css

```diff
@@ -28,9 +28,7 @@ nav a{
   letter-spacing: 2px;
   }
 
-  h1 p {
-    color: black;
-  }
+
 li a:visited{
   color: white;
 }
```

Comment:
Good work on your style sheet but I suggest you use Hex code instead of color names. using Hex code, you get more choices, it is supported compatible by all browsers compared to using color names.